sound/components/sound_components.mk

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# This Makefile snippet builds the sound RX and TX virtualisers
+# it should be included into your project Makefile
+#
+# NOTES:
+#  Generates sound_virt.elf
+#
+
+
+SND_IMAGES := sound_virt.elf
+
+CFLAGS_sound := -I $(SDDF)/include
```

Comment:
This include should be part of the general SDDF includes already.

Reply:
You can keep CFLAGS_sound around as a way to override flags from the project-level Makefile; but it doesn't have to be anything special.

Reply:
Done